src/mappings/helpers/helpers.ts

```diff
@@ -226,9 +226,9 @@ export function createOrLoadProvision(indexerAddress: Bytes, verifierAddress: By
     provision.maxVerifierCutPending = BigInt.fromI32(0)
     provision.thawingPeriod = BigInt.fromI32(0)
     provision.thawingPeriodPending = BigInt.fromI32(0)
-    provision.queryFeeCut = BigInt.fromI32(0)
-    provision.indexingFeeCut = BigInt.fromI32(0)
-    provision.indexingRewardsCut = BigInt.fromI32(0)
+    provision.queryFeeCut = BigInt.fromI32(1000000)
+    provision.indexingFeeCut = BigInt.fromI32(1000000)
+    provision.indexingRewardsCut = BigInt.fromI32(1000000)
```

Comment:
Do we want to move all delegation cuts to the legacy version? Or do we keep Provision level ones lined up with Horizon, and Indexer level ones lined up with Legacy?

I think the latter is better, since it allows the UIs to move to "Stage 2"-like cuts or keep Legacy cuts by using different entities, and keeping both as Legacy just means that once we move to a more generic network subgraph (Stage 2) we'll have extra work that needs to be timed correctly